content/docs/custom-plugins/configuring/index.md

```diff
@@ -0,0 +1,77 @@
+---
+title: Configuring Custom Plugins
+lastUpdated: '2021-06-17T12:11:00.0Z'
+description: How to configure a custom Backstage plugin to Roadie
+---
+
+## Introduction
+
+Roadie supports all standard Backstage plugin types that can be built using instructions defined in [Backstage documentation][backstage-plugin-documentation]. Your plugins can be published via npm or yarn, like publishing a normal package. We will provide you functionality to automatically add custom plugins from repositories to your Roadie instance. If you want to have a private hosted repository for your Roadie plugins with automatic update workflow you can follow instructions on [setting up private secure Artifactory for Roadie plugins.](/docs/custom-plugins/artifactory/)
+
+## Prerequisites
+
+Custom plugins need to be enabled for your Roadie instance. You can enable this by contacting us.
+
+## Configuring a plugin for your Roadie instance
+
+### Step 1. Create plugin definition in Roadie
+
+You can navigate to https://your-company.roadie.so/administration/custom-plugins where you will find a collection of cards detailing custom plugins configured for the Roadie instance. 
+
+![custom_plugins_page](custom_plugins_page.png)
+
+### Form
+
+You can create a new custom plugin definition by clicking "Add New Plugin" floating button. 
+![custom_plugin_form](custom_plugin_form.png)
+
+The values that need to be added are as follows:
+
+##### Package name
+Package field corresponds to the name of the NPM package. This can be a public NPM package hosted by Microsoft in npmjs.com or a private package securely hosted in Roadie Artifactory. If the package is hosted in Roadie Artifactory, the name of the package needs to start with `@<your-company>-roadie/`.
```

Comment:
```suggestion
Package field corresponds to the name of the NPM package. This can be a public NPM package hosted in npmjs.com or a private package securely hosted in Roadie Artifactory. If the package is hosted in Roadie Artifactory, the name of the package needs to start with `@<your-company>-roadie/`.
```

The fact that Microsoft owns npmjs.com is not relevant.